app/src/main/res/values/forms.xml

```diff
@@ -1,1000 +1,966 @@
-  <!--Giratina-->
-  <item>187</item> <!-- Altered Form -->
-  <item>225</item> <!-- Origin Form -->
-  <!--Shaymin-->
-  <item>210</item> <!-- Land Form -->
-  <item>261</item> <!-- Sky Form -->
-  <!--Arceus-->
-  <item>238</item> <!-- Normal Form -->
-  <item>238</item> <!-- Fighting Form -->
-  <item>238</item> <!-- Flying Form -->
-  <item>238</item> <!-- Poison Form -->
-  <item>238</item> <!-- Ground Form -->
-  <item>238</item> <!-- Rock Form -->
-  <item>238</item> <!-- Bug Form -->
-  <item>238</item> <!-- Ghost Form -->
-  <item>238</item> <!-- Steel Form -->
-  <item>238</item> <!-- Fire Form -->
-  <item>238</item> <!-- Water Form -->
-  <item>238</item> <!-- Grass Form -->
-  <item>238</item> <!-- Electric Form -->
-  <item>238</item> <!-- Psychic Form -->
-  <item>238</item> <!-- Ice Form -->
-  <item>238</item> <!-- Dragon Form -->
-  <item>238</item> <!-- Dark Form -->
-  <item>238</item> <!-- Fairy Form -->
-  <!--Basculin-->
-  <item>189</item> <!-- Red Striped Form -->
-  <item>189</item> <!-- Blue Striped Form -->
-  <!--Darumaka-->
-  <item>153</item> <!-- Normal Form -->
-  <item>153</item> <!-- Galarian Form -->
-  <!--Darmanitan-->
-  <item>263</item> <!-- Standard Form -->
-  <item>243</item> <!-- Zen Form -->
-  <item>263</item> <!-- Galarian Standard Form -->
-  <item>323</item> <!-- Galarian Zen Form -->
-  <!--Deerling-->
-  <item>115</item> <!-- Spring Form -->
-  <item>115</item> <!-- Summer Form -->
-  <item>115</item> <!-- Autumn Form -->
-  <item>115</item> <!-- Winter Form -->
-  <!--Sawsbuck-->
-  <item>198</item> <!-- Spring Form -->
-  <item>198</item> <!-- Summer Form -->
-  <item>198</item> <!-- Autumn Form -->
-  <item>198</item> <!-- Winter Form -->
-  <!--Frillish-->
-  <item>115</item> <!-- Normal Form -->
-  <item>115</item> <!-- Female Form -->
-  <!--Jellicent-->
-  <item>159</item> <!-- Normal Form -->
-  <item>159</item> <!-- Female Form -->
-  <!--Stunfisk-->
-  <item>144</item> <!-- Normal Form -->
-  <item>144</item> <!-- Galarian Form -->
-  <!--Tornadus-->
-  <item>266</item> <!-- Incarnate Form -->
-  <item>238</item> <!-- Therian Form -->
-  <!--Thundurus-->
-  <item>266</item> <!-- Incarnate Form -->
-  <item>295</item> <!-- Therian Form -->
-  <!--Landorus-->
-  <item>261</item> <!-- Incarnate Form -->
-  <item>289</item> <!-- Therian Form -->
-  <!--Kyurem-->
-  <item>246</item> <!-- Normal Form -->
-  <item>310</item> <!-- White Form -->
-  <item>310</item> <!-- Black Form -->
-  <!--Keldeo-->
-  <item>260</item> <!-- Ordinary Form -->
-  <item>260</item> <!-- Resolute Form -->
-  <!--Meloetta-->
-  <item>250</item> <!-- Aria Form -->
-  <item>269</item> <!-- Pirouette Form -->
-  <!--Genesect-->
-  <item>252</item> <!-- Normal Form -->
-  <item>252</item> <!-- Douse Form -->
-  <item>252</item> <!-- Shock Form -->
-  <item>252</item> <!-- Burn Form -->
-  <item>252</item> <!-- Chill Form -->
- </integer-array>
+ <!-- Rattata -->
+ <!-- Rattata -->
+ <!-- Rattata -->
```

Comment:
Why does the name appear three times?

Reply:
Yep, probably forgot to change the object name when copy pasting things around